dnf-behave-tests/features/steps/cmd.py

```diff
@@ -40,10 +40,22 @@ def handle_reposync(expected, found):
     return expected, found
 
 
+def run_with_args(context, cmd, **run_args):
```

Comment:
`run_with_args` actually isn't a good name, what are the args? The args are arguments to the `run` function, not the arguments of the command itself (which is what this formulation is otherwise used for in the CI stack context).

`run_in_cwd`, `run_in_work_dir` or perhaps just `run_in_context` as there is the faketime thing as well as the directory?